products/products.html

```diff
@@ -0,0 +1,205 @@
+    <link rel="stylesheet" href="/products/products.css">
+    <title>Pustakalaya | Products</title>
+</head>
+
+<body>
+    <article class="app-container">
+        <header class="nav">
+            <nav class="nav-left">
+                <h2 id="nav-title">Pustakalya</h2>
+                <a class="gray-title" href="#">
+                    <h4" id="nav-title">Home</h4>
+                </a>
+                <a class="gray-title" href="/products/products.html">
+                    <h4" id="nav-title">Shop Now</h4>
+                </a>
+            </nav>
+            <div class="align-center">
+                <div class="search-bar"><input type="search" name="email" placeholder="Search" class="input-rounded" />
+                    <button class="link-no-style search-bar-btn">
+                        <i style="font-size: 1.5rem;" class="bi bi-search"></i>
+                    </button></div>
+            </div>
+            <div class="icon-container">
+                <a class="topnav-link" href=""><span><i class="bi bi-person-fill"></i></span></i></a>
+                <a class="topnav-link" href=""><span><i class="bi bi-suit-heart-fill"></i></span"></a>
+                <a class="topnav-link" href=""><span><i class="bi bi-cart-check-fill"></i></span></a>
+            </div>
+        </header>
+
+
+        <aside class="product-side-bar">
+            <div class="filter-btn">
+                <p class="bar-heading">Filters</p>
+                <p class="bar-heading">Clear</p>
+            </div>
+
+            <div class="price-range-bar">
+                <div class="rangeslider">
+                    <p class="bar-heading">Price</p>
+                    <input type="range" min="100" max="1000" value="10" class="myslider" id="sliderRange">
+                    <p>Value: <span id="demo">12</span></p>
+                </div>
+            </div>
+
+            <div class="side-bar-category">
+                <p class="bar-heading">Category</p>
+                <div>
+                    <input type="checkbox" name="religious" id="religious">Religious
+                </div>
+                <div>
+                    <input type="checkbox" name="fiction" id="fiction">Fiction
+                </div>
+                <div>
+                    <input type="checkbox" name="science" id="science">Science
+                </div>
+                <div>
+                    <input type="checkbox" name="thriller" id="thriller">Thriller
+                </div>
+            </div>
+
+            <div class="ratings">
+                <p class="bar-heading">Ratings</p>
+                <div><input type="radio" name="best" id="best-ratings"> 4 & above</div>
+                <div><input type="radio" name="better " id="better-ratings"> 3 & above</div>
+                <div><input type="radio" name="good" id="good-ratings"> 2 & above</div>
+                <div><input type="radio" name="worst" id="worst-ratings"> 1 & above</div>
+            </div>
+
```

Comment:
for the rating, all radio buttons are selectable, a radio is supposed to be used when a single item is to be selected from a group.
Give all items the same id and it should work.